tests/test_predictions.py

```diff
@@ -20,6 +20,18 @@
     "https://raw.githubusercontent.com/vincentarelbundock/Rdatasets/master/csv/ggplot2/diamonds.csv"
 )
 
+def test_newdata_balanced():
+    df = pl.read_csv(
+        "https://vincentarelbundock.github.io/Rdatasets/csv/palmerpenguins/penguins.csv",
```

Comment:
Is there a "clean" way to save a few datasets like this one locally so we can run the tests without being connected to the internet? I'll open a new issue and tag you.